### ISSUE TITLE
No 3D ResNet50 pretrained model to test

### ISSUE THREAD

**Issue by DonaldKam** [facebookresearch/DetectAndTrack#69]
Thanks for the codes first！And I have a few questions:
1.How can i get the 3D ResNet50 pretrained model to test on the  3D ResNet50, as is mentioned in the paper：“Replacing ResNet-50 in Mask R-CNN with Resnet-101 gave us about 2% improvement in MOTA”.
2.As i found “e2e_keypoint_rcnn_R-101-FPN_1x_trainedCOCO.pkl” ,which is imported as Train Weights（in 2D yaml）in the pretrained model file，I wonder whether it is for 2D training or 3D training？If it is for 2D training，where is the 3D pretrained model for ResNet 101？
3.In 3D yamls files，there are options：CONV_BODY and ROI_HEAD. If I used ResNet 101，I found that there is No ROI_HEAD function for ResNet101. 
If I choose "ResNet3D.add_ResNet101_conv5_body" for CONV_BODY, what should i do with ROI_HEAD(error occurs when CONV_BODY is "ResNet3D.add_ResNet101_conv5_body" and ROI_HEAD option is null)?

**Comment by rohitgirdhar** [comment]
3D experiments were done with ResNet 18. The line from the paper is with regards to 2D models where we experiment with R-50/101.
